src/borg/archiver.py

```diff
@@ -2161,12 +2161,14 @@ def do_break_lock(self, args, repository):
             This pattern style is useful to match whole sub-directories. The pattern
             `pp:root/somedir` matches `root/somedir` and everything therein. A leading
             path separator is always removed.
+            Another example : `pp:foo` will match `foobar`
```

Comment:
can you please test examples before adding them?

Reply:
Sorry, I thought I had tested them but seems to be a hallucination (I am schizophrenic…)